src/MoBi.Core/Commands/RenameObjectBaseCommand.cs

```diff
@@ -39,15 +39,20 @@ protected override void ClearReferences()
       protected override void ExecuteWith(IMoBiContext context)
       {
          base.ExecuteWith(context);
+         RenameBuildingBlock(context);
```

Comment:
this feels weird. Why a RenameBuildingBlock method in a generic objectBaseCommand